plugins/parsers/value/parser.go

```diff
@@ -46,7 +47,7 @@ func (v *ValueParser) Parse(buf []byte) ([]telegraf.Metric, error) {
 		return nil, err
 	}
 
-	fields := map[string]interface{}{"value": value}
+	fields := map[string]interface{}{v.ValueField: value}
```

Comment:
You need to handle the default i.e. the user didn't specify anything...
```suggestion
	fieldname := "value"
	if v.ValueField != "" {
		fieldname = v.ValueField
	}
	fields := map[string]interface{}{fieldname: value}
```